### PR TITLE
Implement ONLY this slice exactly as the issue specifies:

### PR DESCRIPTION
**Pull request by iggyghub**
Implement ONLY this slice exactly as the issue specifies:

# [IPO trading] IPO2: RiskManager.check_order honors a per-call risk-pct override

## What's needed

`RiskManager.check_order` (`cerebral/trading/risk_limits.py`, ~line 55-86) always enforces
the single global `max_per_trade_risk_pct` setting. A prerequisite slice (a separate issue,
land first) adds a nullable `risk_override_pct` field to `StrategySpec` -- this slice makes
`check_order` actually honor a per-call override when the caller supplies one, defaulting to
today's global-only behavior when it doesn't.

## The fix

Add a new optional parameter to `check_order`'s signature (~line 55-63):

```python
def check_order(
    self,
    account_equity: float,
    current_positions_count: int,
    current_daily_loss: float,
    trade_value: float,
    symbol: str,
    qty: float,
    max_per_trade_risk_pct_override: Optional[float] = None,
) -> RiskEvaluation:
```

(`Optional` -- check whether it's already imported at the top of this file; add
`from typing import Optional` if not.)

In the per-trade risk check (~line 74-86), use the override when given instead of the global
config value:

```python
        # 2. Per-trade risk (max loss = % of account)
        effective_pct = (
            max_per_trade_risk_pct_override
            if max_per_trade_risk_pct_override is not None
            else self._config.max_per_trade_risk_pct
        )
        max_per_trade_loss = account_equity * (effective_pct / 100.0)
```

Update the rest of that block's `reason` f-string (currently reads
`f"...exceeds {self._config.max_per_trade_risk_pct}%..."`) to use `effective_pct` instead, so
a blocked-order alert reports the cap that was actually applied, not always the global one.

Do not change anything else in `check_order` -- the max-concurrent-positions check (#1) and
daily-loss check (#3, if present below the per-trade check) are unrelated to this override and
must stay reading the global config exactly as they do today.

## Test

Add a test in `cerebral/tests/test_trading_risk.py` asserting: `check_order` called with
`max_per_trade_risk_pct_override=25.0` allows a trade sized at ~25% of account equity that the
global 10% cap alone would have blocked, and `check_order` called with no override argument
(or `None`) behaves identically to today -- confirm an existing passing test for the global
cap still passes unmodified. Run `python -m pytest cerebral/tests/test_trading_risk.py -q` and
confirm green before committing.

Tests: FAIL

```
........................................................................ [  1%]
........................................................................ [  2%]
........................................................................ [  3%]
........................................................................ [  5%]
........................................................................ [  6%]
........................................................................ [  7%]
........................................................................ [  9%]
........................................................................ [ 10%]
........................................................................ [ 11%]
........................................................................ [ 12%]
........................................................................ [ 14%]
........................................................................ [ 15%]
........................................................................ [ 16%]
........................................................................ [ 18%]
........................................................................ [ 19%]
........................................................................ [ 20%]
........................................................................ [ 22%]
........................................................................ [ 23%]
........................................................................ [ 24%]
........................................................................ [ 25%]
........................................................................ [ 27%]
........................................................................ [ 28%]
........................................................................ [ 29%]
........................................................................ [ 31%]
...........ss........................................................... [ 32%]

```

### CODE REVIEW

**Comment by iggyghub** [comment]
Not merging this PR as-is: its branch picked up an unrelated commit (48b646f, a HELP.md driver file from a separate concurrent self_dev campaign that was never pushed to origin) alongside the real IPO2 change. Cherry-picked just the actual slice commit (ddb06ca) directly onto master instead. Closing without merge.
